src/kaleidoscope/layers.cpp

```diff
@@ -175,21 +163,34 @@ void Layer_::updateActiveLayers(void) {
   // 0). Likewise, for any address where all active layers have a transparent
   // entry, that key will be mapped from the base layer, even if the base layer
   // has been deactivated.
+
```

Comment:
Is there a reason to leave this code in, but commented out?

Reply:
I'd love it if we had a better pattern for debugging printfs. even if it's just a macro.

Reply:
Oops! Thanks for catching that. I have found the Arduino-Trace plugin you introduced to be useful, but sometimes I don't use it out of habit, and a few times I found that it doesn't work. I'll fix this directly.